app-i18n/imsettings/imsettings-1.7.2.ebuild

```diff
@@ -49,18 +52,17 @@ src_prepare() {
 
 	default
 	eautoreconf
-	xdg_environment_reset
 }
```

Comment:
`xdg-utils` seemed to be used just for `xdg_environment_reset` and not anything else, I think we can remove it altogether.